### PR TITLE
Add deprecation info.

### PR DESCRIPTION
**Pull request by hajoeichler**
@lauraluiz, @schleichardt, @stmeissner and @mmoelli please review.

### CODE REVIEW

**Comment by schleichardt** [comment]
:+1: 

But customers might ask about a migration path from old to new...

**Comment by hajoeichler** [comment]
Good point! Do we have one?

**Comment by schleichardt** [comment]
Not yet.
I think it would take two days for me writing a litte documentation how to migrate.

File: README.md [hunk 1]
```diff
@@ -5,6 +5,20 @@
 
 [SPHERE.IO](http://sphere.io) is the first Platform-as-a-Service solution for eCommerce.
 
+## Deprecation notice
+
+We gained a lot of experiences and learnings with this SDK for our SPHERE.IO platform.
+To be transparent, let's list some drawbacks
+- maintainability: The testability wasn't proper ensured.
+- faster development: Integration new endpoints required a lot of code changes.
```
Comment (lauraluiz) [inline README.md]: *"The integration of new endpoints" or "Integrating new endpoints".
Or we can be more generic and say "Extending the SDK with new or custom functionality required a lot of code changes (or rather it was simply impossible)."

File: README.md [hunk 2]
```diff
@@ -5,6 +5,20 @@
...
+- independency: The preselection of the Play framework and other libraries restrict it to a smaller user group.
```
Comment (lauraluiz) [inline README.md]: *restricted it

**Comment by stmeissner** [comment]
:+1:

**Comment by schleichardt** [comment]
Can we then merge it? What about a migration documentation? Should we start one? Or just write for now we will do such a thing?

**Comment by emmenko** [comment]
I would at least add a "coming soon" migration guide

**Comment by hajoeichler** [comment]
Will add a note and we can hopefully add the donut store as example later.

**Pull request merged**